run.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -e
+
+if [ ! -f ./ircd.yaml ]; then
+  cp oragono.yaml ircd.yaml
+fi
+
+if [ ! -f ircd.db ]; then
+  ./oragono initdb
+fi
+
+if [ ! -f tls.crt ]; then
+  ./oragono mkcerts
+fi
+
+./oragono run
```

Comment:
Ahh I see, that makes sense. I was expecting to start up a docker machine and then do it that way, haven't really done packaging apps the `docker run` way before. This looks like it works fine, thanks for the help with it!

I've pushed this change to the [`devel+docker` branch](https://github.com/oragono/oragono/compare/master...devel+docker)